gravitee-reporter-elasticsearch-http/src/main/java/io/gravitee/reporter/elastic/config/ElasticConfiguration.java

```diff
@@ -115,6 +134,13 @@ public String getIndexName() {
 		return endpoints;
 	}
 
+    private List<String> initializeiIngestPlugins() {
+        String ingestPluginsSt = environment.getProperty("reporters.elasticsearch.ingestPlugins", String.class);
+        return ingestPluginsSt != null ? Pattern.compile(",").splitAsStream(ingestPluginsSt)
```

Comment:
Why not using `Stream.of(str.split(","))` instead ?

Reply:
For me, it's the same. I thought that "Pattern.compile(",").splitAsStream(", it was little bit more efficiency. it's as you like

Reply:
We do not need high efficiency at this level since it will be run only once. So you can keep as-it, no problem.